plugins/rocksdb/src/main/java/org/hyperledger/besu/plugin/services/storage/rocksdb/segmented/RocksDBColumnarKeyValueStorage.java

```diff
@@ -150,42 +149,41 @@ private BlockBasedTableConfig createBlockBasedTableConfig(final RocksDBConfigura
   }
 
   @Override
-  public AtomicReference<ColumnFamilyHandle> getSegmentIdentifierByName(
-      final SegmentIdentifier segment) {
+  public RocksDbSegmentIdentifier getSegmentIdentifierByName(final SegmentIdentifier segment) {
```

Comment:
is this modification voluntary to revert what did @garyschulte on this PR ? https://github.com/hyperledger/besu/pull/3734/files#diff-1b1c62528a9b69821bd095065e34f149b9be0f6abdc301d86e7b9dbf14fc3318R154

Reply:
yes, this revert the use of AtomicReference<S> in the interface, and keeps the storage plugin interface stable